trunk/SUAVE/Plots/Geometry_Plots/plot_vehicle.py

```diff
@@ -39,286 +40,582 @@ def plot_vehicle(vehicle, save_figure = False, plot_control_points = True, save_
+    wake_edge_color = 'blue' 
+    wake_alpha      = 0.5  
+    if'Wake' in VD:
+        plot_propeller_wake(axes, VD,wake_face_color,wake_edge_color,wake_alpha)
+        
+    # -------------------------------------------------------------------------
+    # PLOT FUSELAGE
+    # -------------------------------------------------------------------------        
+    fuselage_face_color = 'grey'                
+    fuselage_edge_color = 'black' 
+    fuselage_alpha      = 1      
+    for fus in vehicle.fuselages:     
+        plot_fuselage(axes, fus ,fuselage_face_color,fuselage_edge_color,fuselage_alpha)     
+  
+    # -------------------------------------------------------------------------
+    # PLOT ENGINE
+    # -------------------------------------------------------------------------        
+    propulsor_face_color = 'darkred'                
+    propulsor_edge_color = 'black' 
+    propulsor_alpha      = 1    
+    for propulsor in vehicle.propulsors:    
+        plot_propulsor(axes,VD,propulsor,propulsor_face_color,propulsor_edge_color,propulsor_alpha)    
+      
+    # Plot Vehicle
+    plt.axis('off') 
+    plt.grid(None)      
+    return 
+
+def plot_wing(axes,VD,face_color,edge_color,alpha_val): 
+    """ This plots the wings of a vehicle 
+
+    Assumptions: 
+    None
+
+    Source:   
+    None
+    
+    Inputs:   
+    VD                   - vortex distribution    
+    
+    Properties Used:
+    N/A
+    """      
     
     n_cp = VD.n_cp 
     for i in range(n_cp): 
+        
         X = [VD.XA1[i],VD.XB1[i],VD.XB2[i],VD.XA2[i]]
         Y = [VD.YA1[i],VD.YB1[i],VD.YB2[i],VD.YA2[i]]
         Z = [VD.ZA1[i],VD.ZB1[i],VD.ZB2[i],VD.ZA2[i]] 
+        
         verts = [list(zip(X, Y, Z))]
+        
         collection = Poly3DCollection(verts)
         collection.set_facecolor(face_color)
         collection.set_edgecolor(edge_color)
         collection.set_alpha(alpha_val)
+        
         axes.add_collection3d(collection)     
-        max_range = np.array([VD.X.max()-VD.X.min(), VD.Y.max()-VD.Y.min(), VD.Z.max()-VD.Z.min()]).max() / 2.0    
+        max_range = np.array([VD.X.max()-VD.X.min(), VD.Y.max()-VD.Y.min(), VD.Z.max()-VD.Z.min()]).max() / 2.0 
+        
         mid_x = (VD.X .max()+VD.X .min()) * 0.5
         mid_y = (VD.Y .max()+VD.Y .min()) * 0.5
         mid_z = (VD.Z .max()+VD.Z .min()) * 0.5
+        
         axes.set_xlim(mid_x - max_range, mid_x + max_range)
         axes.set_ylim(mid_y - max_range, mid_y + max_range)
         axes.set_zlim(mid_z - max_range, mid_z + max_range)    
         
-    if  plot_control_points:
-        axes.scatter(VD.XC,VD.YC,VD.ZC, c='r', marker = 'o' )
+    return    
  
+def plot_propeller_wake(axes, VD,face_color,edge_color,alpha):  
+    """ This plots a helical wake of a propeller or rotor
+
+    Assumptions: 
+    None
+
+    Source:   
+    None
+    
+    Inputs:   
+    VD                   - vortex distribution    
```

Comment:
be more specific

Reply:
More specific headers made. Issue Resolved